docs/sources/next/javascript-api/k6-experimental/webcrypto/subtlecrypto/importkey.md

```diff
@@ -73,3 +75,150 @@ export default async function () {
 ```
 
 {{< /code >}}
+
+### Import a static raw key and decrypt transmitted data
+
+This example demonstrates how to import a static `raw` key and decrypt some transmitted data (which contains initialization vector and encoded data) in `base64`.
+
+{{< code >}}
+
+```javascript
+import { crypto } from 'k6/experimental/webcrypto';
+import { b64decode } from 'k6/encoding';
+
+export default async function () {
+  // transmitted data is the base64 of the initialization vector + encrypted data
+  // that unusually transmitted over the network
```

Comment:
Ooh I see. In that case, maybe we could reword it to: "this is data usually transmitted over the network".

Or include response body there as well as an example: "this is data usually transmitted over the network, such as a response body"

Reply:
Now, I'm wondering if we should drop that part :sweat_smile: 

Actually, doesn't matter how data is received, essential details (in this example) that transmitted data is a concatenation of an initialization vector and encrypted data; some people could use this approach, that's why I used it to bring examples more real-world close

Reply:
That's a fair point! Actually, what if we added this context on line 81? Something along the lines of:

"This example demonstrates how to import a static `raw` key and decrypt some transmitted data in `base64`. The transmitted data in this example represents an initialization vector and encoded data, and in a real-world scenario, it can be a response body or other data received from a request."

It would keep the code sample concise but still give context to users.

Reply:
Adjusted :heavy_check_mark: